frontend/src/metabase/css/query_builder.css

```diff
@@ -489,6 +489,19 @@
   border-right: none;
 }
 
+.NativeQueryEditorDragHandle {
+  background: var(--color-border);
```

Comment:
I think we'll want to darken this up a bit, it's a little subtle right now even on a reasonably high quality laptop screen so I'd imagine it'll be all but imperceptible on displays with less resolution and contrast. I *think* the drag handle cursor is enough to indicate interaction but I might also consider a small bg hover state.

**Currently**
<img width="1280" alt="Screen Shot 2019-11-25 at 10 23 58 AM" src="https://user-images.githubusercontent.com/5248953/69553170-b9b94480-0f6d-11ea-97ee-e4d0d04d2212.png">

```suggestion
  background: color(var(--color-border) blackness(+ 3%));
```

**A smidge darker**
<img width="1280" alt="Screen Shot 2019-11-25 at 10 33 49 AM" src="https://user-images.githubusercontent.com/5248953/69554096-46183700-0f6f-11ea-90fa-7816894d0eb2.png">